components/trade/Orderbook.tsx

```diff
@@ -104,15 +102,18 @@ const Orderbook = () => {
         element.scrollTop = (element.scrollHeight - element.offsetHeight) / 2
       }
     }
-  }, [])
+  }, [orderbookElRef])
```

Comment:
is this dependency needed? i don't think my editor or eslint complain about refs being required hook dependencies, but maybe they are?

Reply:
Is it not best practice to include all of the props used in the callback as dependencies?

Reply:
I checked and variables from useRef do not need to be included in the dependency array of the useEffect. 

"This is because useRef returns a mutable ref object whose .current property is initialized with the passed argument. The object returned from useRef will persist for the full lifetime of the component."